src/models/UsersModel.js

```diff
@@ -0,0 +1,51 @@
+class UsersModel {
+    constructor({
```

Comment:
I don't see any associations in these models. Since we aren't using SQL (where I would create a table that just matches a user ID to a course ID, we instead probably want to create associations of a user to a course in the user object itself (a user can be enrolled in one or more courses).

Reply:
Initially I commented this code. UsersModel is associated with userController                                                                                                                         "const UsersModel = require('../models/UsersModel')"                          
 const user = new UsersModel({
            name,
            email,
            profilePicture: 'gs://learninghub-ggpacteam3.appspot.com/images/userProfileImage.jpg' // Use your default image URL
          });